tools/spelling/spelling_dictionary.txt

```diff
@@ -358,6 +358,7 @@ Wi
 XDS
 XFCC
 XFF
+XFH
```

Comment:
optional (since we do this for XFF) but it might be worth just spelling this one out at the 3 places we mention XFH since it's not as well known as XFF.